server/backend/database/memory/housekeeping_test.go

```diff
@@ -28,38 +28,38 @@ import (
 	"github.com/stretchr/testify/assert"
 	monkey "github.com/undefinedlabs/go-mpatch"
 
-	"github.com/yorkie-team/yorkie/server/backend/database"
 	"github.com/yorkie-team/yorkie/server/backend/database/memory"
 )
 
 func TestHousekeeping(t *testing.T) {
 	memdb, err := memory.New()
 	assert.NoError(t, err)
-	projectID := database.DefaultProjectID
 
 	t.Run("housekeeping test", func(t *testing.T) {
 		ctx := context.Background()
 
-		yesterday := gotime.Now().Add(-24 * gotime.Hour)
+		_, project, err := memdb.EnsureDefaultUserAndProject(ctx, "test", "test")
+		assert.NoError(t, err)
+
+		yesterday := gotime.Now().Add(-25 * gotime.Hour)
```

Comment:
Have you had any problems here?

Reply:
`findDeactivateCandidatesPerProject` inside `FindDeactivateCandidates()` needs `project` object to find deactivation candidates based on `project`'s `clientDeactivateThreshold`.

But previous test only provided `project ID`, not the whole `project`. So EnsureDefaultUserAndProject() was needed to get proper `project`.